src/tools/annotation/EllipticalRoiTool.js

```diff
@@ -235,7 +239,7 @@ export default class EllipticalRoiTool extends BaseAnnotationTool {
 
         const ellipseOptions = { color };
 
-        if (renderDashed) {
+        if (renderDashed || data.dashedBorder) {
```

Comment:
`dashedBorder` is a bit ambiguous, should be `hasDashedBorder` to indicate it's a boolean by starting with a verb